src/windowmanagement/WindowLayout/WindowLayoutProvider.tsx

```diff
@@ -75,11 +68,39 @@ export function WindowLayoutProvider({ children }: { children: React.ReactNode }
     }
   }
 
+  function closeWindow(id: string) {
+    if (!rcDocRef.current) {
+      return;
+    }
+
+    const existingPanel = rcDocRef.current.find(id);
+    if (existingPanel) {
+      rcDocRef.current.dockMove(existingPanel as TabData | PanelData, null, 'remove');
+    }
+  }
+
+  function createWindowTabData(
+    id: string,
+    title: string,
+    content: React.JSX.Element
+  ): TabData {
+    return {
+      id,
+      title,
+      content: <ScrollArea h={'100%'}>{content}</ScrollArea>,
+      closable: true,
+      cached: true,
+      group: 'regularWindow'
+    };
+  }
+
```

Comment:
This seems like a pure function to me. Could we make this a helper function and not have it as a part of this provider?

Reply:
We could.  The reason I put it here for now is that the use case for me felt similar to the `addWindow` function, for example, and that it directly controls the code of the windows that the provider creates. So to me, it made sense to have it in the same class. 

Would be good to discuss where to place it, more precisely :)